{{cookiecutter.repostory_name}}/envs/dev/.env.template

```diff
@@ -38,3 +38,7 @@ CSP_MANIFEST_SRC="{{cookiecutter.csp_manifest_src}}"
 CSP_WORKER_SRC="{{cookiecutter.csp_worker_src}}"
 CSP_BLOCK_ALL_MIXED_CONTENT={{cookiecutter.csp_block_all_mixed_content}}
 CSP_EXCLUDE_URL_PREFIXES={{cookiecutter.csp_exclude_url_prefixes}}
+BACKUP_B2_BUCKET=
```

Comment:
should this be defined in a dev environment?

Reply:
Why not, could be useful.

Reply:
Doesn't hurt, could be useful.

Reply:
In my opinion, would be perfect if we can preserve the same set of env variables in all of our environments if it's possible.
Our tools should be prepared to skip something in case e.g. `dev` environment whether by setting an empty variable or by checking `ENV` variable that I introduced in the past.